src/server/symbols.map

```diff
@@ -144,6 +144,7 @@ MIR_SERVER_2.9 {
     mir::Server::override_the_session_listener*;
     mir::Server::override_the_shell*;
     mir::Server::override_the_window_manager_builder*;
+    mir::Server::override_the_default_window_manager_builder*;
```

Comment:
Wrong stanza. If we need this function (I don't think we do) then it belongs in `MIR_SERVER_2.14`. (The stanza is part of our ABI management and indicates the Mir release the function is introduced in.)

Reply:
I don't think we need this function any longer, but I am curious to learn a little bit more about this `symbols.map` file during our chat today. I noticed that I couldn't link with out implementing this line

Reply:
Look at the end of the file for functions we're adding to the next release

Reply:
Understood! And this file is updated by hand? Or is it generated?

Reply:
Looks like I found the answer [here](https://mir-server.io/doc/dso_versioning_guide.html)